src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/BookingTransformer.kt

```diff
@@ -44,6 +45,8 @@ class BookingTransformer(
     originalArrivalDate = jpa.originalArrivalDate,
     originalDepartureDate = jpa.originalDepartureDate,
     createdAt = jpa.createdAt.toInstant(),
+    turnaround = jpa.turnaround?.let { turnaroundTransformer.transformJpaToApi(it) },
```

Comment:
No big deal, but believe you can do a method reference here too like you have on the next line?